sprint-issue-template.md

```diff
@@ -0,0 +1,32 @@
+# Sprint <% date %>
+
+## Sprint Goals
+
+- 
+
+## Sprint Discussions
+
+[Schedule](https://github.com/ipfs/pm#sprint-discussion-schedule)
+
+Please take notes in a separate pad, if you can, and link it here.
+
+Endeavour         | Lead           | Pad    | Permanent Link
+:---------------: | :------------: | :----: |
+Apps on IPFS      | Juan Benet     |        |
```

Comment:
Sounds good, though I think one time slot for both would be enough for now and if it expands we can split it into two time slots.